### PR TITLE
Blogpost on EKS

### PR DESCRIPTION
**Pull request by donkersgoed**

### CODE REVIEW

File: _includes/header.html [hunk 1]
```diff
@@ -15,7 +15,6 @@
             <span class="icon-bar"></span>
           </button>
           <a class="navbar-brand" href="/">
-            <img src="https://sentia.com/static/svg/logo.svg" height="60" alt="Sentia" />
```
Comment (donkersgoed) [inline _includes/header.html]: this image doesn't exist anymore

Comment (donkersgoed) [reply _includes/header.html]: also, the website will get a makeover soon, so no need to fix it now.

File: _posts/2018-10-21-Integrating-EKS-with-other-AWS-services.md [hunk 2]
```diff
@@ -0,0 +1,794 @@
...
+
+Re:Invent 2018 is being held in Las Vegas next week. We expect that AWS will announce more suported regions for EKS, leading to more EKS deployments in the near future. We hope this blog post will help you set up the integrations and complex configurations for all your use cases.
+
+The challenges we will be discussing fall into two categories:
+- Reliably integrating HTTP frontend services (CloudFront, API Gateway and WAF) with EKS
+- Using Infrastructure as Code (CloudFormation) to deploy resources that depend on EKS managed resources
+
+## About EKS
+Elastic Container Service for Kubernetes (somehow abbreviated to EKS) is Amazon's implementation of a managed Kubernetes service. In a nutshell, Amazon provides the master nodes, you provide the worker nodes, you do some configuration, and voilà: you have a highly available, scalable, and relatively cheap Kubernetes cluster.
```
Comment (kbessas) [inline _posts/2018-10-21-Integrating-EKS-with-other-AWS-services.md]: You mention EKS above but you only explain here the abbreviation. Typically this is done in the first occurrence in the text.

File: _posts/2018-10-21-Integrating-EKS-with-other-AWS-services.md [hunk 3]
```diff
@@ -0,0 +1,794 @@
+---
+layout: post
+title: Integrating EKS with other AWS Services
+banner: /assets/posts/2018-10-21-Integrating-EKS-with-other-AWS-services/banner.png
+author: lvandonkersgoed
+
+---
+EKS offers developers an easy way to run Kubernetes workloads at AWS. But what if you need to integrate your EKS based app with other services like CloudFront, API Gateway or Web Application Firewall?
+
+For a recent project we needed to deploy a number of environments with EKS clusters. Each of these environments had different requirements regarding the supporting services; some needed API Gateway, some needed WAF, others needed CloudFront, and many required a combination of these services. We've worked to create a "one size fits all" deployment for EKS - a configuration that allows for integration with all of these services. You can find our results below.
```
Comment (kbessas) [inline _posts/2018-10-21-Integrating-EKS-with-other-AWS-services.md]: other needed

File: _posts/2018-10-21-Integrating-EKS-with-other-AWS-services.md [hunk 4]
```diff
@@ -0,0 +1,794 @@
...
+
+The challenges we will be discussing fall into two categories:
+- Reliably integrating HTTP frontend services (CloudFront, API Gateway and WAF) with EKS
+- Using Infrastructure as Code (CloudFormation) to deploy resources that depend on EKS managed resources
+
+## About EKS
+Elastic Container Service for Kubernetes (somehow abbreviated to EKS) is Amazon's implementation of a managed Kubernetes service. In a nutshell, Amazon provides the master nodes, you provide the worker nodes, you do some configuration, and voilà: you have a highly available, scalable, and relatively cheap Kubernetes cluster. 
+
+Some EKS specific features:
+- The workers nodes for EKS are provided by EC2 instances in an autoscaling group. Fargate support for EKS is expected to be released soon.
```
Comment (kbessas) [inline _posts/2018-10-21-Integrating-EKS-with-other-AWS-services.md]: worker nodes

**Pull request merged**